src/feed/subscription/RealtimeSubscription.ts

```diff
@@ -22,17 +22,20 @@ class RealtimeSubscription extends Subscription {
     }
 
     async _startSubscribe(tickHistoryRequest: TCreateTickHistoryParams) {
+        const contract_info = this.contractInfo as ProposalOpenContract;
         //here we include duration = 'ticks' && exclude duration = 'seconds' which hasn't tick_stream, all_ticks, tick_count (consist of 15-86.400 ticks)
-        if(!this.shouldFetchTickHistory && !!(this.contractInfo as ProposalOpenContract).tick_stream){
+        if(!this.shouldFetchTickHistory && !!contract_info.tick_stream){
             const [tickHistoryPromise, processTickHistory] = this._getProcessTickHistoryClosure();
-            this._binaryApi.subscribeTickHistory(Object.assign(tickHistoryRequest, { count: (this.contractInfo as ProposalOpenContract).tick_count} ), processTickHistory);
+            this._binaryApi.subscribeTickHistory(Object.assign(tickHistoryRequest, { count: contract_info.tick_count} ), processTickHistory);
             const response = await tickHistoryPromise;
             const quotes = this._processHistoryResponse(response);
             this._tickCallback = processTickHistory;
             return { quotes, response };
         }else{
```

Comment:
@akmal-deriv i know this is not your change but could you please run option_shift_f in this file to prettify it ?